drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -137,4 +137,20 @@ def self.sort_by_option(option)
   end
   use_enum :ethnicity_enum_map, ::HUD.ethnicities.slice(0, 1), include_base_null: true
   use_common_enum :veteran_status_enum_map, :no_yes_reasons
+
+  def age(date = Date.current)
+    return unless self.DOB.present?
+
+    date = date.to_date
+    dob = self.DOB.to_date
+    self.class.age(date: date, dob: dob)
+  end
+
+  def self.age(date:, dob:)
+    return nil unless date.present? && dob.present?
+
+    age = date.year - dob.year
+    age -= 1 if dob > date.years_ago(age)
+    age
+  end
```

Comment:
Let's just depend on the logic we already have in `GrdaWarehouse::Hud::Client`
```suggestion
  def age(date = Date.current)
    GrdaWarehouse::Hud::Client.age(date: date, dob: self.DOB)
  end

  def self.age(date:, dob:)
    GrdaWarehouse::Hud::Client.age(date: date, dob: self.DOB)
  end
```